server/lib/settings-location-update.js

```diff
@@ -14,20 +14,13 @@ const geocoder = nodeGeocoder({
  * Hook a method to construct the geolocation of the address
  */
 export async function updateLocationSettings(id){
-
   const query = Settings.findById(id)
   const settings = await query.select('+gmapsApiKey +gmapsClientId')
 
   const address = settings.address
-
   const [result] = await geocoder.geocode(address)
-
-  if (!result) return next(new ValidationError({
-    fields: Object.assign({}, ...addressFields.map((field, i) => ({
-      [field.meta]: i === 0 ? 'Address not found' : ' '}
-    )))
-  }))
-
-  const {latitude, longitude} = result
-  await Settings.findByIdAndUpdate(id, {'location': {'lat': latitude, 'lng': longitude}})
+  if (result) {
```

Comment:
I meant there should still be a ValidationError if no result is returned, but there are no `fields` in the settings model to show it on. So the error should point to something that is in the settings model. Where would you want to show the error if the address doesn't geocode?